man/docker-daemon.8.md

```diff
@@ -74,6 +77,16 @@ format.
 **--bip**=""
   Use the provided CIDR notation address for the dynamically created bridge (docker0); Mutually exclusive of \-b
 
+**--cluster-store**=""
+  URL of the distributed storage backend
+
+**--cluster-advertise**=""
+  Specifies the 'host:port' combination that this particular daemon instance should use when advertising
+  itself to the cluster. The daemon should be reachable by remote hosts on this 'host:port' combination.
```

Comment:
fwiw, this one seems a little strange to me, since I was trying to convey the user is responsible for picking the correct IP address/port that is reachable, but I've made the change you suggest.